mtest/testscript/scripts/text_tab_preserve_placement.script

```diff
@@ -0,0 +1,12 @@
+init init/Treble.mscx
+cmd note-input
+cmd note-input
+palette Fingering style Fingering text 1
+cmd flip
+cmd staff-text
+cmd flip
+cmd next-element
+cmd system-text
+cmd flip
+cmd next-element
```

Comment:
Seems like the second staff text & system text are recorded, but not the second fingering...